INSTALL

```diff
@@ -191,6 +195,11 @@ Note 4: You need to use a POSIX (or nearly POSIX) make. GNU make
   is not guaranteed to work. Don't be surprised if the first thing
   that make does is call configure again.
 
+Note 5: Ptl gets installed in the parent directory of where pbs is installed.
```

Comment:
PTL is an acronym. Please capitalize it.
Also, it may be better to say PTL uses the same prefix as PBS.